Update tagline (#9)
zitadel/.github: profile/README.md

    <img src="https://user-images.githubusercontent.com/9879976/200943036-2fea2cba-454f-447a-a203-bc318c93fbdf.svg#gh-dark-mode-only" alt="Zitadel Logo" width="300px" height="auto" />
</p>

ZITADEL - Identity infrastructure, simplified for you.

> An open souce identity infrastructure, built for the serverless era, acting as drop-in multi-tenancy auth solution with self-service for you b2b needs.

Reviewer: ```suggestion
The open source identity infrastructure for developers, acts as drop-in multi-tenancy auth solution with self-service for all your B2B needs.
```

Reviewer: How about this instead?
